planning/obstacle_avoidance_planner/include/obstacle_avoidance_planner/common_structs.hpp

```diff
@@ -138,6 +139,9 @@ struct DebugData
   boost::optional<geometry_msgs::msg::Pose> stop_pose_by_drivable_area = boost::none;
   std::vector<geometry_msgs::msg::Point> interpolated_points;
   std::vector<geometry_msgs::msg::Point> straight_points;
+  std::vector<geometry_msgs::msg::Point> left_bound;
+  std::vector<geometry_msgs::msg::Point> right_bound;
```

Comment:
Drivable area's rviz marker is published by behavior_path_planner node.
Therefore, I think we do not need to add debug marker here.